### PR TITLE
docs: initial pt translation

### LINKED ISSUE #5548: [HacktoberFest][Documentation] Diffusers documentation to Portuguese
Hi!

I am opening this issue to encourage people translating this project, and other AI knowledge to Portuguese.
I would like to contribute to this too, specially because of this year's hacktoberfest 🎃.

For this, you'll need:

- copy english docs from [source/en](https://github.com/huggingface/diffusers/blob/main/docs/source/en/) to source/pt
- Translate the files you copied
- Add and translate [_toctree.yml](https://github.com/huggingface/diffusers/blob/main/docs/source/en/_toctree.yml) to pt one
- Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue.
    For hacktoberfest 🎃, please also do claim this issue by writing "I am solving this issue in <link-to-pr>"

### Get started

- [x] [index.md](https://github.com/huggingface/diffusers/blob/main/docs/source/en/index.md) @SirMonteiro
- [x] [quicktour.md](https://github.com/huggingface/diffusers/blob/main/docs/source/en/quicktour.md) @SirMonteiro
- [x] [installation.md](https://github.com/huggingface/diffusers/blob/main/docs/source/en/installation.md) @SirMonteiro

### PR DESCRIPTION
**Pull request by SirMonteiro**
# What does this PR do?

Fixes #5548

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/pt/quicktour.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+```
+
+Para acelerar o processo de retirada de ruído, mova a entrada e o modelo para uma GPU:
+
+```py
+>>> model.to("cuda")
+>>> noisy_sample = noisy_sample.to("cuda")
+```
+
+Now create a denoising loop that predicts the residual of the less noisy sample, and computes the less noisy sample with the scheduler:
```
Comment (stevhliu) [inline docs/source/pt/quicktour.md:280]: This sentence should also be translated

**Review by stevhliu — COMMENTED** [review]
Thanks for kickstarting the Portuguese translations! 🎉 🇵🇹 

To build the docs, can you add the language code to this file:

https://github.com/huggingface/diffusers/blob/0d4b459be6f7d43fba6b976f859f38a67f3e82af/.github/workflows/build_documentation.yml#L19

and this one:

https://github.com/huggingface/diffusers/blob/0d4b459be6f7d43fba6b976f859f38a67f3e82af/.github/workflows/build_pr_documentation.yml#L18

File: docs/source/pt/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+A Biblioteca tem três componentes principais:
+
+- Pipelines de última geração para a geração em poucas linhas de código. Têm muitos pipelines no 🤗 Diffusers, veja a tabela no pipeline [Visão geral](api/pipelines/overview) para uma lista completa de pipelines disponíveis e as tarefas que eles resolvem.
+- Intercambiáveis [agendadores de ruído](api/schedulers/overview) para balancear as compensações entre velocidade e qualidade de geração.
+- Pretrained [models](api/models) that can be used as building blocks, and combined with schedulers, for creating your own end-to-end diffusion systems.
```
[the remark this answers is not in the document]
Comment (SirMonteiro) [reply docs/source/pt/index.md]: Thanks!, I forgot to remove base text from this part.

File: docs/source/pt/installation.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+```bash
+pip install diffusers["flax"] transformers
+```
+</jax>
+</frameworkcontent>
+
+## Instalação a partir do código fonte
+
+Antes da instalação do 🤗 Diffusers a partir do código fonte, certifique-se de ter o PyTorch e o 🤗 Accelerate instalados.
+
```
[the remark this answers is not in the document]
Comment (SirMonteiro) [reply docs/source/pt/installation.md:56]: This exactly sentence is saying to make sure that PyTorch and Accelerate is already installed.

File: docs/source/pt/quicktour.md (hunk 1 above)
Comment (SirMonteiro) [reply docs/source/pt/quicktour.md]: Thanks, this sentence is translated now!

Comment (SirMonteiro) [reply docs/source/pt/quicktour.md]: Ok, I added pt to both GitHub workflow files!
can you add a "hacktoberfest" label for the related issue, please?

**Comment by SirMonteiro** [comment]
Ok, I added pt to both GitHub workflow files!
can you add a "hacktoberfest" label for the related issue, please?
